app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
On `gem yank`, info cache is cleared [here](https://github.com/rubygems/rubygems.org/pull/1384/files#diff-175d93318045eb4395c87409e05dc6cfR57). No need to do it again.